src/slskd/Integrations/Scripts/ScriptService.cs

```diff
@@ -115,7 +115,7 @@ private async Task HandleEvent(Event data)
                                 CreateNoWindow = true,
                                 RedirectStandardOutput = true,
                                 RedirectStandardError = true,
-                                Arguments = run.Command.StartsWith(DefaultCommandPrefix) ? run.Command : $"{DefaultCommandPrefix} {run.Command}",
+                                Arguments = run.Command.StartsWith(DefaultCommandPrefix) ? run.Command : $"{DefaultCommandPrefix} \"{run.Command}\"",
```

Comment:
Wouldn't this break if the user supplied a command which includes double quotes? ie:

```sh
touch "/tmp/notify me"
```

Reply:
it didn't when i was testing on Linux last night.  i'm going to test on Windows this morning

Reply:
Broke on Windows; I pushed a fix.  Thanks for pointing it out.